memory-bank/tasks.md

```
@@ -5,4 +5,170 @@
+
+### Success Criteria
+
+1. ✅ Docusaurus site loads correctly at <https://texarkanine.github.io/a16n/>
+2. ✅ Docs deploy immediately for prose-only changes
+3. ✅ Docs deployment skipped when code changes are included
+4. ✅ Docs automatically rebuild after package releases
+5. ✅ Single docs build per release cycle (not N builds for N packages)
+
+### Implementation Complete
+
+**PR Created:** https://github.com/Texarkanine/a16n/pull/20
```

markdownlint-cli2 (0.20.0)

[warning] Bare URL used

(MD034, no-bare-urls)